pkg/platform/abstract/platform.go

```diff
@@ -480,6 +480,14 @@ func (ap *Platform) ValidateFunctionConfig(ctx context.Context, functionConfig *
 
 func (ap *Platform) AutoFixConfiguration(ctx context.Context, err error, functionConfig *functionconfig.Config) bool {
 	if errors.RootCause(err).Error() == "V3IO Stream trigger does not support autoscaling" {
+		if *functionConfig.Spec.MinReplicas == 0 {
+			ap.Logger.WarnWithCtx(ctx, "V3IO Stream doesn't support scaling from zero - "+
+				"Auth fixing by setting minReplicas to 1",
```

Comment:
It doesn't support autoscaling, but are you sure it doesn't support scale from zero?

Reply:
@TomerShor here is a condition we throw the error:
```
if maxReplicas != nil && *minReplicas != *maxReplicas {
					return nuclio.NewErrBadRequest("V3IO Stream trigger does not support autoscaling")
				}
```
which means that we don't support scaling from/to zero as well. Actually, we do not support scaling to zero in any stream trigger, because we don't have a dlx mechanism for those.